beta/nncf/tensorflow/sparsity/magnitude/algorithm.py

```diff
@@ -160,8 +146,8 @@ def _set_masks_for_threshold(self, threshold_val):
             for weight_attr, ops in wrapped_layer.weights_attr_ops.items():
                 weight = wrapped_layer.layer_weights[weight_attr]
 
-                for op_name, op in ops.items():
-                    if isinstance(op, BinaryMask):
+                for op_name in ops.keys():
```

Comment:
Can you please refactor this according to the style guide?

Reply:
Done